packages/console/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@logto/console",
+  "version": "0.1.0",
+  "description": "> TODO: description",
+  "author": "Gao Sun <gao@silverhand.io>",
+  "homepage": "https://github.com/logto-io/logto#readme",
+  "license": "MPL-2.0",
+  "scripts": {
+    "preinstall": "npx only-allow pnpm",
+    "precommit": "lint-staged",
+    "start": "parcel src/index.html",
+    "check": "tsc --noEmit",
+    "build": "pnpm check && rm -rf dist && parcel build src/index.html --no-autoinstall",
+    "lint": "eslint --ext .ts --ext .tsx src",
+    "stylelint": "stylelint \"src/**/*.scss\""
+  },
+  "dependencies": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "devDependencies": {
+    "@parcel/core": "^2.3.1",
+    "@parcel/transformer-sass": "^2.3.1",
+    "@silverhand/eslint-config-react": "^0.8.0",
+    "@silverhand/ts-config-react": "^0.8.0",
+    "@types/react": "^17.0.14",
+    "@types/react-dom": "^17.0.9",
+    "eslint": "^8.1.0",
+    "lint-staged": "^11.1.1",
+    "parcel": "^2.3.1",
+    "postcss": "^8.4.6",
+    "postcss-modules": "^4.3.0",
+    "prettier": "^2.3.2",
+    "stylelint": "^13.13.1",
+    "typescript": "^4.3.5"
```

Comment:
Should we use typescript `4.5.5`? It's been there for a while now

Reply:
let's upgrade the workspace typescript version in a separate PR

Reply:
Sure, np